pkgs/tools/typesetting/tex/pplatex/default.nix

```diff
@@ -0,0 +1,49 @@
+{
+	pkgs,
```

Comment:
Please indent with two spaces instead of a tab. Please have a look at other files in nixpkgs to understand what our style looks like.